File: lib.rpiws281x/src/main/java/com/externalalarmclock/lib/rpiws281x/ERpiWs281xStripType.java
package com.externalalarmclock.lib.rpiws281x;

import com.externalalarmclock.rpiws281x.RpiWs281xLibrary;

public enum ERpiWs281xStripType {
	SK6812_STRIP_GRBW(RpiWs281xLibrary.SK6812_STRIP_GRBW);
	
	private final int libStripType;

	private ERpiWs281xStripType(int libStripType) {
		this.libStripType = libStripType;
	}

	public int getLibStripType() {
		return libStripType;
	}
}

File: lib.rpiws281x/src/main/java/com/externalalarmclock/lib/rpiws281x/ERpiWs281xStripType.kt
package com.externalalarmclock.lib.rpiws281x

import com.externalalarmclock.rpiws281x.RpiWs281xLibrary

enum class ERpiWs281xStripType private constructor(val libStripType: Int) {
    SK6812_STRIP_GRBW(RpiWs281xLibrary.SK6812_STRIP_GRBW)
}
